### ISSUE TITLE
cerebral infarct

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3418]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

cerebral infarct

>2. Synonyms (e.g., absent spleen)
 
brain infarct, cerebrum infarct

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

a necrotic lesion in the cerebrum resulting from a sudden insufficiency of arterial or venous blood supply due to emboli, thrombi or mechanical factors 

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

https://jamanetwork.com/journals/jamaneurology/fullarticle/776636
DOI: 10.1001/archneur.57.6.869

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

brain lesion, abnormal cerebral hemisphere morphology

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
review existing infarction size terms these should be infarct not infarction

**Comment by sbello** [comment]
MP:0021003

**Comment by anna-anagnostop** [comment]
Per Sue's comment:

Reopened ticket to review existing infarction size terms and change decreased/increased cerebral infarction size and decreased/increased myocardial infarction size to use **infarct** rather than infarction in the term label.

infarct is (pathology) an area of dead tissue caused by a loss of blood supply; a localized necrosis while infarction is (pathology) the process which causes an infarct (see https://wikidiff.com/infarct/infarction)
